docs/g10-criterion.md

```diff
@@ -372,29 +373,35 @@ decision left to the health IT developer, as long as the “refreshed” refresh
+	After an access token expires, an app can subsequently connect to an authorization server to exchange a valid refresh token for a new access token and also have its refresh token renewed without needing to obtain end-user authorization. During both exchanges, security is increased (i.e., greater protection against leaked refresh tokens) when confidential apps use their client secret for client authentication. The (g)(10) criterion paragraphs at § 170.315(g)(10)(v)(A)(1)(ii) and § 170.315(g)(10)(v)(A)(2)(ii) require that apps “capable of storing a client secret” and thus capable of authenticating themselves, must be given a refresh token upon valid first time connections and have their refresh tokens renewed upon valid subsequent connections. This enables persistent access for apps capable of storing a client secret without requiring end-user re-authorization.
 
+	**Subsequent connections:**
 	``` mermaid
 		sequenceDiagram
-			participant app as App capable of <br/> storing a client secret
+			participant app as App
 			participant authz as Health IT Module's Authorization Server
 			participant fhir as Health IT Module's FHIR® Server
 
 			loop while refresh token is valid
-			app ->> authz: Refresh token
-			note over app,authz: Client secret used for client authentication
-			note over authz: The (g)(10) criterion paragraph at § 170.315(g)(10)(v)(A)(2)(ii) <br/> requires that applications capable of storing a client secret have their <br/> refresh tokens renewed (i.e., made valid for a new three month period). <br/> This could include the authorization server issuing a new refresh token <br/> or renewing the initial refresh token.
-			alt Health IT Module's Authorization Server issues a new refresh token
-				authz -->> app: New access token and new refresh token response
-			else Health IT Module's Authorization Server renews existing refresh token
-				authz -->> app: New access token response
-				authz -->> authz: Existing refresh token renewed
-			end
-			loop while new access token is valid
-				app ->> fhir: New access token used to request resources
-			end
+				alt App is capable of storing a client secret
+					app ->> authz: Refresh token
+					note over app,authz: Client secret used for authentication
+					note over authz: The (g)(10) criterion paragraph at § 170.315(g)(10)(v)(A)(2)(ii) <br/> requires that apps capable of storing a client secret have their <br/> refresh tokens renewed (i.e., made valid for a new three month period). <br/> This could include the authorization server issuing a new refresh token <br/> or renewing the initial refresh token.
```

Comment:
Suggest revising "(i.e., made valid for a new three month period)" to "(i.e., made valid for a new period of no less than three months)"